docs/src/Masonry.doc.js

```diff
@@ -68,6 +68,13 @@ card(
         description:
           'A callback when the user scrolls and you need to load more items into the grid. Note that `scrollContainer` must be specified.',
       },
+      {
+        name: 'layoutDirection',
+        type: 'string',
```

Comment:
I would treat this one as `dealDirection = 'up' | 'right' | 'down' | 'left'` in Tooltip, accepting  `ltr` or `rtl` rather than just `string`. https://pinterest.github.io/gestalt/#/Tooltip